tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -150,6 +150,7 @@ def __init__(
       trainable=True,
       checkpoint=True,
       init_size=0,
+      kv_creator=None,
```

Comment:
`Variable` is decorated as a TrackableResource. But there is no holding resource inside this class. This will make save/load failure in TF2. Will it be possible to let the `kv_creator` creates a `resource` and let it manage the storage backend, such as `CuckooHashTable`, or `redis`, etc., since there might be some kv is not explicitly specified by devices.